src/css/index.scss

```diff
@@ -0,0 +1,105 @@
+section.contact {
+  padding-bottom: 100px;
+  padding-top: 10px;
+}
+section.contact h2 {
+  font-size: 38px;
+  font-size-adjust: inherit;
+  font-family: avenir, sans-serif;
+  font-style: normal;
+  margin-bottom: 2%;
+  margin-top: 20px;
+  padding-left: 10%;
```

Comment:
From my perspective, I don't suggest the percentage margin or padding, as the content is fixed size. We should keep the position and UI consistent.

Reply:
The reason I use the percentage is that if the web broswer change size, the content will change as well.

Reply:
Do I have to make all content consistent on the web page?

Reply:
Alex I get what you mean, it is quite rare for us to use percentage based padding/margin since we using media query to switch to different styles base on screen size. We think that give us better control over on how it looks on different devices. But what you are doing is not wrong, just personal preference. Keep in mind sometimes this is the designer's call on how layout behaves on different screen sizes.